source/Octopus.Tentacle/Kubernetes/Scripts/RunningKubernetesJob.cs

```diff
@@ -410,25 +411,44 @@ void RecordScriptHasStarted(IScriptLogWriter writer)
 
         void RecordScriptHasCompleted(int exitCode)
         {
+            using var writer = ScriptLog.CreateWriter();
             try
             {
                 var scriptState = stateStore.Load();
                 scriptState.Complete(exitCode);
                 stateStore.Save(scriptState);
+                WriteInfo(writer, $"Job '{jobName}' has completed with exit code {exitCode}");
```

Comment:
I think this one can be `Verbose`
```suggestion
                WriteInfo(writer, $"Kubernetes Job '{jobName}' completed with exit code {exitCode}");
```